src/dmd/objc_glue.d

```diff
@@ -54,6 +54,12 @@ ObjcGlue objc()
 // Should be an interface
 extern(C++) abstract class ObjcGlue
```

Comment:
This doesn't require C++ linkage anymore does it?

Any backend support in gdc/ldc won't be using this module IIUC.

Reply:
> This doesn't require C++ linkage anymore does it?

No.